tests/Dockerfile/FedoraDockerfile.in

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Usage: FROM [image name]
+FROM @OS_DISTRIBUTION@
+
+ENV TEST_REPO="github.com/kata-containers/tests"
```

Comment:
can you change `TEST_REPO` to `TESTS_REPO`. I remember we agreed to always use "tests" to avoid issues.